src/containers/GroupOverview/components/GroupItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { Group } from 'types/Types';
+
+import MembersIcon from '@material-ui/icons/PersonRounded';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.background.default,
+    border: '1px solid #aaa',
+    borderRadius: 10,
+    cursor: 'pointer',
```

Comment:
Istedenfor å gjøre en div clickable så bør du heller bruke for eks et <Card /> fra Material UI som har ripple effekt og faktisk er en knapp. Der er viktig for universell utforming og at det blir likt som resten av siden. Andre alternativer er ListItem og Button